apps/fluent-tester/src/E2E/common/BasePage.ts

```diff
@@ -18,6 +31,27 @@ export class BasePage {
       timeout ?? this.waitForPageTimeout,
       this._pageName + ' did not render correctly. Please see /errorShots of the first failed test for more information.',
       1000,
+      // {
+      //   timeout: timeout ?? this.waitForPageTimeout,
```

Comment:
Are these commented lines for later debugging?

Reply:
Yes, once I upgrade these packages, this change will be needed. I removed the upgrade from this PR though because I ran into some issues. I should have it sorted out by next week though :)

Reply:
if leaving in for future re-use, add a usage comment? :)